spec/services/importers/party_data_spec.rb

```diff
@@ -4,7 +4,7 @@
   describe '#perform(oscn_id)' do
     it 'saves aliases' do
       path = 'spec/fixtures/party_example.html'
-      html = File.open(path).read
+      html = File.read(path)
```

Comment:
did you mean to commit this?

Reply:
Again i think this is a rubocop thing that auto corrects whenever I run it. it seems to prefer the `File.read(path)` over  'File.open(path).read' .